packages/firestore/test/unit/specs/spec_test_components.ts

```diff
@@ -66,12 +67,17 @@ export class MockMemoryPersistence extends MemoryPersistence {
     ) => PersistencePromise<T>
   ): Promise<T> {
     if (this.injectFailures) {
-      return Promise.reject(
-        new IndexedDbTransactionError(new Error('Simulated retryable error'))
-      );
-    } else {
-      return super.runTransaction(action, mode, transactionOperation);
+      if (this.injectFailures[action as PersistenceAction] === undefined) {
```

Comment:
It looks like the Android and iOS spec test framework will need to be updated to handle this change in the generated JSON. Are you planning to do that as well?

Reply:
Yes, we need to make those changes there too. For now, the "recovery" tests are marked `no-ios` and `no-android`. We won't need to update the runner until we remove these tags and implement LevelDB/SQLite recovery.

Reply:
Ahh you're right. I missed the `no-ios` and `no-android` tags... even though I was looking for them 🤔